src/inflection.js

```diff
@@ -69,6 +85,24 @@ class Inflection {
     return this.languageCode
   }
 
+  /**
+   * Sets grammar properties based on inflection info
+   */
+  setGrammar () {
```

Comment:
It must be called only when all inflection features are populated. Inflection object is created without them initially. Those features are populated within a morph adapter library one by one, as morphological analyzer response is parsed. Only after that process is complete, `setGrammar()` can be called.

The other reason is that I did not want to change tufts adapter for now. The changes already made to `data models` and `inflection tables` are complex enough to make merge harder than usual, so I thought it's better not to touch other libraries as this will create more dependencies and increase complexity even more. But I think it should be called somewhere within a morphological adapter so other libraries would receive those properties fully populated. What do you think?